src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/data/apps/tdd/endring-av-navn/ui/FormLayout.json

```diff
@@ -0,0 +1,378 @@
+{
+  "data": {
```

Comment:
A lot of the files in the app can be excluded wrt these tests I think.. 
We would need the config files, but do we need anything else?

Reply:
i have removd some